test/testinput/bufr_ncep_prepbufr_adpupa.py

```diff
@@ -53,50 +53,54 @@ def test_bufr_to_ioda(DATA_PATH, OUTPUT_PATH, date):
 
     # Use the ResultSet returned to get numpy arrays of the data
     # MetaData
-    cat = r.get('prepbufrDataLevelCategory', 'prepbufrDataLevelCategory')
-    lat = r.get('latitude', 'prepbufrDataLevelCategory')
-    lon = r.get('longitude', 'prepbufrDataLevelCategory')
+    cat = r.get('prepbufrDataLevelCategory')
+    lat = r.get('latitude')
+    lon = r.get('longitude')
     lon[lon>180] -= 360  # Convert Longitude from [0,360] to [-180,180]
-    sid = r.get('stationIdentification', 'prepbufrDataLevelCategory')
-    elv = r.get('stationElevation', 'prepbufrDataLevelCategory')
-    tpc = r.get('temperatureEventCode', 'prepbufrDataLevelCategory')
-    pob = r.get('pressure', 'prepbufrDataLevelCategory')
+    sid = r.get('stationIdentification')
+    sid = np.tile(sid, (lon.shape[1],1))
+    elv = r.get('stationElevation')
+    tpc = r.get('temperatureEventCode')
+    pob = r.get('pressure')
     pob *= 100
 
     # Time variable
-    hrdr = r.get('timeOffset', 'prepbufrDataLevelCategory')
+    hrdr = r.get('timeOffset')
     print("cycleTimeSinceEpoch")
     cycleTimeSinceEpoch = np.int64(calendar.timegm(time.strptime(date, '%Y%m%d%H%M')))
     hrdr = np.int64(hrdr*3600)
     hrdr += cycleTimeSinceEpoch
```

Comment:
@emilyhcliu @PraveenKumar-NOAA @rmclaren Could you clarify what this variable is? When using a 00Z prepbufr file, the times in this variable end up being ~21Z

Reply:
@haydenlj is profile level cycle-time, which is used for dateTime calculation. 

| HRDR     | 004218 | PROFILE LVL TIME-CYCLE (FOR RAOB/PIBAL, BASED ON B DFT)  |

You can find more details in the following table: 
https://github.com/JCSDA-internal/ioda-converters/blob/develop/doc/bufr_table_samples/adpupa.prepbufr.table.txt

Reply:
@haydenlj @emilyhcliu for 00Z prepbufr file, the cycle starts at 21Z and end ends at 03Z.